policyengine_canada/variables/gov/provinces/qc/tax/income/credits/cce/cce_tax_credit.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_canada.model_api import *
+
+
+class cce_tax_credit(Variable):
```

Comment:
All provincial variables start with the code
```suggestion
class qc_cce_tax_credit(Variable):
```